File: QuanLyPhongKham.Server/src/QuanLyPhongKham.DTOs/Responses/DanhMucDichVuResponse.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace QuanLyPhongKham.DTOs.Responses
{
    public class DanhMucDichVuResponse
    {
        public Guid MaDV { get; set; }
        public string TenDV { get; set; }
        public decimal DonGia { get; set; }
    }
}

File: QuanLyPhongKham.Client/src/interfaces/responses/DanhMucDichVuResponse.js
/**
 * @typedef {Object} DanhMucDichVuResponse
 * @property {string} maDV
 * @property {string} tenDV
 * @property {number} donGia
 */

export const createDanhMucDichVuResponse = (
  maDV = "",
  tenDV = "",
  donGia = 0,
) => ({
  maDV,
  tenDV,
  donGia,
});
